lib/storage/src/content_manager/consensus_manager.rs

```diff
@@ -12,6 +12,7 @@ use collection::collection_state;
 use collection::common::is_ready::IsReady;
 use collection::shards::shard::PeerId;
 use collection::shards::CollectionId;
+pub use common::defaults::CONSENSUS_DEFAULT_META_OP_WAIT as DEFAULT_META_OP_WAIT;
```

Comment:
```suggestion
pub use common::defaults;
```

---

I'd refactor usage to explicit `defaults::CONSENSUS_META_OP_WAIT`.